docs/developer-docs/integrations/independently-verifying-ic-signatures.md

```diff
@@ -0,0 +1,125 @@
+# Independently verifying Internet Computer signatures
+
+## Overview
+
+Calls on the IC require that a cryptographic identity be attached to the call. This identity can be anonymous or authenticated using a cryptographic signature. Canisters can use this attached identity to respond to a call, which enables contracts to use those identities for other workflows and purposes. 
```

Comment:
Technically, I'd think that an `identity` includes both the public and private keys. Since the private key stays private, what we actually send along is a signature signed by the `identity`, along with a public key, encoded as a `Principal`. 

The same goes for delegations, except we need to provide the Delegation Chain as well, so that we can prove the delegations allow the "signing" public key to sign on behalf of the `sender`.